### PR TITLE
New 8085 instructions cheat sheet

### PR DESCRIPTION
**Pull request by jophab**
**What does your Instant Answer do?**
Shows 8085 instructions cheat sheet
**What is the data source for your Instant Answer? (Provide a link if possible)**
http://home.iitk.ac.in/~ajalan/8085_is_details.pdf
**Why did you choose this data source?**
It reliable
**Are there any other alternative (better) data sources?**
Yes may be
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
students
**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
nope
**Are you having any problems? Do you need our help with anything?**
no
**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

-----
IA Page: https://duck.co/ia/view/8085_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/8085.json [hunk 1]
```diff
@@ -0,0 +1,230 @@
+{
+    "id": "8085_cheat_sheet",
+    "name": "8085",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/8085.json]: Let's change this to `8085 Instruction Set` (we should do the same for 8086)

File: share/goodie/cheat_sheets/json/8085.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    "description": "8085 instructions and their explanations",
+    "metadata": {
+        "sourceName": "iitk",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/8085.json]: Capitalize this: `IITK`

File: share/goodie/cheat_sheets/json/8085.json [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+        "sourceUrl": "http://home.iitk.ac.in/~ajalan/8085_is_details.pdf"
+    },
+    "template_type": "reference",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/8085.json]: Are these Code snippets? Or entered on a command line? the `code` or `terminal` templates may be more appropriate.

I think this is OK though

**Comment by jophab** [comment]
@moollaza okay?

**Pull request closed without merging**